master.sh

```diff
@@ -178,7 +214,20 @@ start_k8s(){
 		fi
 		sleep 1
 	done
+}
 
+start_k8s(){
+    if [ "$1" == "hyperkube" ]; then
+      start_hyperkube_k8s
+    elif [ "$1" == "service" ]; then
+      start_service_k8s
+    else
+      echo "You should specify how to start kubelet: as a service or in hyperkube."
+      exit 1
+    fi
+}
+
+start_hyperkube_k8s(){
```

Comment:
Small comment: I would change the name to indicate close relations:
start_k8s
start_k8s_hyperkube   or   start_k8s_as_hyperkube 
start_k8s_service         or   start_k8s_as_service